launch/package.xml

```diff
@@ -17,6 +17,7 @@
   <test_depend>ament_flake8</test_depend>
   <test_depend>ament_pep257</test_depend>
   <test_depend>python3-pytest</test_depend>
+  <test_depend>python3-mock</test_depend>
```

Comment:
Just wondering, is there any difference between `python3-mock` and `pytest-mock`? (I'm not sure if we have a rosdep key for the later).

Reply:
from what I understand, `pytest-mock` is the pip package and `python3-mock` is from apt, so I used `python3-mock`

Reply:
Based on [pypi docs](https://pypi.org/project/mock/):

> mock is now part of the Python standard library, available as unittest.mock in Python 3.3 onwards.

I guess we can avoid the key and use `unittest.mock` directly.

Reply:
> from what I understand, pytest-mock is the pip package and python3-mock is from apt, so I used python3-mock

They seem to be different packages.
Considering that `unittest.mock` is part of the standard library, I would just use that one.

Reply:
got it; changed in 16dd37a5ae3870a371eb4c3e0361ff624580b57c